src/Classes/ItemsTab.lua

```diff
@@ -2610,18 +2610,26 @@ function ItemsTabClass:AddCrucibleModifierToDisplayItem()
 		return table.concat(label, "/")
 	end
 	local function itemCanHaveMod(mod)
-		local keyMap = { }
+		local keyMap, includeTags = { }, { }
 		for index, key in ipairs(mod.weightKey) do
 			keyMap[key] = index
 		end
+		-- check for uniques with off-tag mods
+		if self.build.data.casterTagCrucibleUniques[self.displayItem.title] then
```

Comment:
isnt data a global variable? you shouldnt need to look at self.build to access it